### PR TITLE
Add title

### PR DESCRIPTION
**Pull request by alinawinter**
Please check

### CODE REVIEW

**Review by zamukaev — APPROVED** [review]
great

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+  font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+    "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+  background-color: #e8f0f7;
```
Comment (alexWaligorski) [inline css/styles.css:5]: I would rather define the default background-color for the body and not with the "all elements"-selector. can't exactly explain why.. maybe a question for the coaches? there is probably a best practice concerning this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <link rel="stylesheet" href="./css/styles.css" />
```
Comment (alexWaligorski) [inline index.html:3]: link should be by convention insode head element: https://www.w3schools.com/CSS/css_howto.asp

**Review by alexWaligorski — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (alinawinter) [reply index.html:3]: aaah thank you!!!

File: css/styles.css (hunk 1 above)
Comment (alinawinter) [reply css/styles.css:5]: aaaah okay! Yes maybe let's ask or later play around and see what changes, when we use it in different parts....

**Pull request merged**